requirements/requirements-dev.in

```diff
@@ -46,5 +47,5 @@ coverage
 bump2version
 
 # Test support
-matrix-synapse==1.27.0
+matrix-synapse==1.29.0
```

Comment:
Matrix 1.27.0 has a problem with certain SQLite versions: https://github.com/matrix-org/synapse/issues/8996
I ran into this problem when running the tests and had to update it locally.

This has been fixed in version 1.28.0, and the latest version seems to be working fine.

Reply:
I could also create a separate PR to upgrade this if you prefer to not have this mixed, but I'd still upgrade the version to avoid a known issue.

Reply:
Yes, please move this to a separate PR. Synapse updates could make a difference in the overall behavior of Raiden and I want to have the update explicitly approved by @ulope or @fredo instead of sneaking it in in this PR. We might also want to align the version with the one used in the servers we run.

Reply:
Done https://github.com/raiden-network/raiden/pull/6876